src/__tests__/downshift.misc-with-utils-mocked.js

```diff
@@ -37,7 +37,7 @@ test('does not scroll from an onMouseMove event', () => {
   }
   const {queryByTestId} = render(<HighlightedIndexController />)
   const input = queryByTestId('input')
-  const item = queryByTestId('item-1')
+  const item = queryByTestId('item-2')
```

Comment:
I managed to 'fix' this test, although I need some more insight on what it tests. I used that example as a story and I still could not figure out how it works and how I managed to break it. Some help, please?

Reply:
This is actually fine. Expected for the change that's been made.